README.md

```diff
@@ -87,6 +87,13 @@ Minimum:
 * Should theoretically work with Zsh 5.4 or newer, but I'm unable to test that.
 
 ## Installing & Updating
+### [Oh-My-Zsh](https://github.com/ohmyzsh/ohmyzsh)
+If you use Oh-My-Zsh, clone the repository inside your oh-my-zsh repo to install:
+```zsh
+git clone https://github.com/marlonrichert/zsh-autocomplete ${ZSH_CUSTOM:-${ZSH:-~/.oh-my-zsh}/custom}/plugins/zsh-autocomplete
```

Comment:
```suggestion
```zsh
git clone --depth=1 --shallow-submodules https://github.com/marlonrichert/zsh-autocomplete ${ZSH_CUSTOM:-${ZSH:-~/.oh-my-zsh}/custom}/plugins/zsh-autocomplete
```zsh
```

No need to pull down the full repo history. 😉 

I don't use OMZ so not sure whether those nested pathname variables are convention, or perhaps better replaced with the basic `~/.oh-my-zsh/custom/plugins/zsh-autocomplete` and point out to change if required.

Reply:
Yes. The variable pathnames are automatically set up by OMZ. I don't think the added flags are necessary. Please see a similar zsh plugin project for examples.(https://github.com/zsh-users/zsh-completions)